src/app/action-confirm-dialog/action-confirm-dialog.component.ts

```diff
@@ -28,7 +28,7 @@ export class ActionConfirmDialogComponent implements OnInit, LoggedInCallback, C
   private conf = AppConf;
   pointsEarned;
   dialog: MatDialog;
-  private actionService = new ActionService(this.dialog);
+ // private actionService = new ActionService(this.dialog);
```

Comment:
Hey just wanted to make sure we are commenting it without breaking anything/

Reply:
Also seems like there are few conflicts to be resolved before you merge.

Reply:
I don't see any conflicts. do you see it? could you please point me to the conflicts where you see it? I just uncommented this line "private actionService = new ActionService(this.dialog);" this line has initialization error so commented out. now it's open.